transitions/registration.js

```diff
@@ -333,14 +340,43 @@ module.exports = {
         var doc = options.doc,
             db = db || options.db;
 
-        transitionUtils.addUniqueId(db, doc, callback);
+        if (options.params.patient_id) {
```

Comment:
I think the param property could be more clearly named. How about `patient_id_field` or something? This would then be consistent with `patient_name_field` at line 378.